components/x-audio/package-lock.json

```diff
@@ -0,0 +1,6798 @@
+{
```

Comment:
`package-lock.json` shouldn't be committed in components, see #158. I think we should be able to prevent this using a top-level `npmrc` file

Reply:
Oh, this should already be happening #200 🤦‍♀